src/app/public/modules/needs-attention/types/action-hub-needs-attention.ts

```diff
@@ -0,0 +1,13 @@
+import { NavigationExtras } from '@angular/router';
+
+export type SkyActionHubNeedsAttention = {
```

Comment:
We will need JSDocs here in order to fully populate the docs.

Example: https://github.com/blackbaud/skyux-lookup/blob/master/src/app/public/modules/country-field/types/country.ts